js/pwdOnOff.js

```diff
@@ -0,0 +1,13 @@
+export const changePwViewMode = (e) => {
+  // pw on off 버튼 이벤트
+  if (e.target.className === "pw-onoff") {
+    const pwdInputTag = e.target.previousElementSibling;
+    if (pwdInputTag.type === "password") {
+      e.target.src = "../imgs/eye-on.svg";
+      pwdInputTag.type = "text";
+    } else {
+      e.target.src = "../imgs/eye-off.svg";
+      pwdInputTag.type = "password";
+    }
+  }
+};
```

Comment:
여기도 early return을 활용하면 코드 보기가 한결 좋아질거 같아요

```suggestion
export const changePwViewMode = (e) => {
  // pw on off 버튼 이벤트
  if (e.target.className !== "pw-onoff") return;
  
  const pwdInputTag = e.target.previousElementSibling;
    
  if (pwdInputTag.type === "password") {
      e.target.src = "../imgs/eye-on.svg";
      pwdInputTag.type = "text";
  } else {
      e.target.src = "../imgs/eye-off.svg";
      pwdInputTag.type = "password";
    }
};
```